README.md

```diff
@@ -36,6 +36,7 @@ The package can be installed as:
     username: "your.name@your.domain", # or {:system, "SMTP_USERNAME"}
     password: "pa55word", # or {:system, "SMTP_PASSWORD"}
     tls: :if_available, # can be `:always` or `:never`
+    tls_versions: [:"tlsv1", :"tlsv1.1", :"tlsv1.2"], # additional details: http://erlang.org/doc/man/ssl.html
```

Comment:
I renamed the option to `allowed_tls_versions`, which felt least ambiguous. Although SMTP-servers might "support" multiple tls versions, we might want to allow only some of them.